src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/utils/TaskMandatoryFieldsValidatorIntegrationTest.java

```diff
@@ -53,19 +57,57 @@ void setUp() {
+        ", 'someTaskType', 'PUBLIC', 'title', 'CASE_ID', 'Asylum', 'CaseCategory', 'CaseName', 'IA', 'TestRegion', '765324', 'roleCategory', 'workTypeResource'",
+        "'someTaskName', , 'PUBLIC', 'title', 'CASE_ID', 'Asylum', 'CaseCategory', 'CaseName', 'IA', 'TestRegion', '765324', 'roleCategory', 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', , 'CASE_ID', 'Asylum', 'CaseCategory', 'CaseName', 'IA', 'TestRegion', '765324', 'roleCategory', 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', 'title', , 'Asylum', 'CaseCategory', 'CaseName', 'IA', 'TestRegion', '765324', 'roleCategory', 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', 'title', 'CASE_ID', , 'CaseCategory', 'CaseName', 'IA', 'TestRegion', '765324', 'roleCategory', 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', 'title', 'CASE_ID', 'Asylum', , 'CaseName', 'IA', 'TestRegion', '765324', 'roleCategory', 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', 'title', 'CASE_ID', 'Asylum', 'CaseCategory', , 'IA', 'TestRegion', '765324', 'roleCategory', 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', 'title', 'CASE_ID', 'Asylum', 'CaseCategory', 'CaseName', , 'TestRegion', '765324', 'roleCategory', 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', 'title', 'CASE_ID', 'Asylum', 'CaseCategory', 'CaseName', 'IA', , '765324', 'roleCategory', 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', 'title', 'CASE_ID', 'Asylum', 'CaseCategory', 'CaseName', 'IA', 'TestRegion', , 'roleCategory', 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', 'title', 'CASE_ID', 'Asylum', 'CaseCategory', 'CaseName', 'IA', 'TestRegion', '765324', , 'workTypeResource'",
+        "'someTaskName', 'someTaskType', 'PUBLIC', 'title', 'CASE_ID', 'Asylum', 'CaseCategory', 'CaseName', 'IA', 'TestRegion', '765324', 'roleCategory', ",
+
+    })
+    @DisplayName("should throw ServiceMandatoryFieldValidationException when a mandatory field is missing")
+    void should_throw_service_mandatory_field_validation_exception_when_a_mandatory_field_is_missing(String taskName, String taskType, String securityClassification, String title,  String caseId, String caseTypeId, String caseCategory, String caseName, String jurisdiction, String region, String location, String roleCategory, String workTypeResource) {
+        TaskResource task = getTaskResource(taskId, taskName, taskType, securityClassification, title, caseId, caseTypeId, caseCategory, caseName, jurisdiction, region, location, roleCategory, workTypeResource);
+        ServiceMandatoryFieldValidationException exception = assertThrows(ServiceMandatoryFieldValidationException.class, () -> taskMandatoryFieldsValidator.validate(task));
+        String message = exception.getMessage();
+        assertTrue(message.contains("cannot be null or empty"));
```

Comment:
could you improve this to include the name of the field that cannot be null or empty?